src/pages/scripting/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: UXP Scripting
+description: Create your first UXP-powered script
+contributors:
+  - https://github.com/amandahuarng
+---
+
+# Overview
+
+UXP scripting allows developers to execute a single file to accomplish tasks in Photoshop. Unlike ExtendScript, these scripts can utilize the [UXP](../uxp/) and [Photoshop](../ps_reference/) APIs to communicate with Photoshop.
+
+## When are scripts useful?
+Scripts are mainly used to perform one-off tasks that end-users would otherwise perform manually. It is especially useful when the task is repetitive in nature, such as adding a watermark to your documents. When a script is invoked by a user, it executes a task and is unloaded by Photoshop once it’s done. If required, a dialog can be used to prompt the user for inputs. Scripts can be executed in a headless manner or with minimal UI, without the need for any manifest setup.
+
+The biggest advantage that scripting offers is its simplicity. You can accomplish tasks in Photoshop without requiring as elaborate of a setup as that of a UXP plugin.  Any script code can be reused in a UXP plugin as well.
+
+## How is it different from UXP plugin development? 
+1. **[Lifetime](../scripting/how-it-works/index.md#execution-context):** A script’s lifetime ends when it finishes executing.
+2. **[Limited UI](../scripting/how-it-works/index.md#user-interface):** Unlike plugins, scripts cannot have a panel UI. They can only create dialog UIs.
+3. **No persistent data:** Scripts also don’t have access to persistent storage like the plugin data folder or local storage. 
```

Comment:
"local storage" sounds like local fs storage, but you just mean `window.localStorage`, right?

Reply:
yes I'll clarify